common/djangoapps/enrollment/tests/test_api.py

```diff
@@ -8,6 +8,8 @@
 from django.test import TestCase
 from django.test.utils import override_settings
 from django.conf import settings
+
+from course_modes.models import CourseMode
```

Comment:
@nasthagiri There are already test cases around explicitly passing the mode.  This case is well covered.